Validation/MtdValidation/plugins/BtlRecHitsValidation.cc

```diff
@@ -96,6 +119,34 @@ void BtlRecHitsValidation::analyze(const edm::Event& iEvent, const edm::EventSet
   const MTDTopology* topology = topologyHandle.product();
 
   auto btlRecHitsHandle = makeValid(iEvent.getHandle(btlRecHitsToken_));
+  auto btlSimHitsHandle = makeValid(iEvent.getHandle(btlSimHitsToken_));
+  MixCollection<PSimHit> btlSimHits(btlSimHitsHandle.product());
+
+  // --- Loop over the BLT SIM hits
+  std::unordered_map<uint32_t, MTDHit> m_btlSimHits;
```

Comment:
@casarsa this energy accumulation is equivalent to that used in the digi - rechit construction, am I correct?

Reply:
Yes, you are. In both cases I'm starting from the SimHits and am accumulating the energy released in the same BTL crystal.